k8sutils/k8sutils.go

```diff
@@ -30,10 +31,16 @@ type leaderElection interface {
 }
 
 // CreateKubeClientSet - Returns kubeclient set
-func CreateKubeClientSet() error {
+func CreateKubeClientSet(kubeConfig ...string) error {
```

Comment:
I would have thought that this argument would just be a string. Is this to avoid code changes where this function is called?